Grow.c

```diff
@@ -3068,6 +3069,39 @@ static void catch_term(int sig)
 	sigterm = 1;
 }
 
+/**
+ * handle_forking() - Handle forking, helper function for reshapes.
+ *
+ * @forked: is process forked.
+ * @devname: device name.
+ * @service: wheather a service should be started or just fork.
+ *
+ * Returns: -1 if failed to fork,
+ *           0 if currently child process,
+ *           1 process forked, passed to service or delegated to udev.
+ */
+static int handle_forking(bool forked, char *devname, bool service)
+{
+	/* Nothing to do if already forked. */
+	if (forked)
+		return 0;
+
+	/* Just fork if no service. */
```

Comment:
this comment is not true, or maybe variable name in not detailed enough. Probably it should be `use_systemd`

Service could be there, we don't want it